src/rrt/Tree.hpp

```diff
@@ -236,8 +279,13 @@ class Tree {
      */
     Node<T>* grow() {
         //  extend towards goal, waypoint, or random state depending on the
-        //  biases
-        //  and a random number
+        //  biases and a random number
+        if (_nodes.size() == 1) {
+            flann::Matrix<double>* root = new flann::Matrix<double>(
+                rootNode()->coordinates(), 1,
+                sizeof(rootNode()->state()) / sizeof((double)(0.0)));
+            _kdtree.buildIndex(*root);
```

Comment:
Do you need to create a variable here just to pass the value into the `buildIndex` function? Would something like `buildIndex( flann::Matrix<double>(...) )` work?